caab-service/src/integrationTest/java/uk/gov/laa/ccms/data/api/service/BaseApplicationServiceIntegrationTest.java

```diff
@@ -360,6 +360,46 @@ public void testGetApplication_noData() {
     assertEquals(HttpStatus.NOT_FOUND, exception.getHttpStatus());
   }
 
+  @Test
+  @Sql(scripts = {
+      "/sql/application_insert.sql",
+      "/sql/linked_cases_insert.sql",
+      "/sql/opponent_insert.sql",
+      "/sql/case_outcome_insert.sql",
+      "/sql/prior_authority_insert.sql",
+      "/sql/proceeding_insert.sql",
+      "/sql/scope_limitation_insert.sql"
+  })
+  public void testRemoveApplication_applicationExists_removesSuccessfully() {
```

Comment:
Do we want to check the existence before and after this test?

Reply:
Good spot. That was a bit of a lame test. Corrected.